### PR TITLE
Sign-off the translations generated by the translation bot

### PR DESCRIPTION
**Pull request by christianlupus**
All commits in all PRs of apps are piped through the DCO checker. Depending on the CI/CD routines of the app, this causes problems with the commits from the translation bot. So, you get a ton of errors when merging changes between different branches. If the need for further explanations is there, I can open an issue to explain the problem in detail.

This PR should make the bot signoff all translation commits from transifex. This is not reduced to the apps but to all here existing integrations. If not desired for an individual integration I can remove it. But I think, it should be consistent.

### CODE REVIEW

File: translations/handleAppTranslations.sh [hunk 1]
```diff
@@ -91,7 +91,7 @@ do
 
   # create git commit and push it
   git add l10n/*.js l10n/*.json
-  git commit -am "[tx-robot] updated from transifex" || true
+  git commit -am "[tx-robot] updated from transifex" s || true
```
Comment (MorrisJobke) [inline translations/handleAppTranslations.sh:94]:
```suggestion
  git commit -am "[tx-robot] updated from transifex" -s || true
```

**Review by MorrisJobke — APPROVED** [review]
Makes sense 👍

**Comment by christianlupus** [comment]
@MorrisJobke What is the process here? Are you merging or is there a detailed review process?

**Comment by MorrisJobke** [comment]
> @MorrisJobke What is the process here? Are you merging or is there a detailed review process?

Sorry - I missed this.

**Comment by christianlupus** [comment]
No problem, @MorrisJobke. I was just unsure.

**Pull request merged**